cmd/info/info.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2024, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package info
+
+import (
+	"fmt"
+	"github.com/gosuri/uitable"
+	"github.com/oracle-cne/ocne/pkg/cmdutil"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+const (
+	CommandName = "info"
+	helpShort   = "Display information about OCNE "
```

Comment:
It would be good to clarify this short help message. Perhaps something like, "Display usage information". Also, I think it is better to omit OCNE. It may be redundant

Reply:
Thanks! I made these changes.